cylc/flow/platforms.py

```diff
@@ -214,9 +229,7 @@ def platform_from_name(
                 'regular expression. See the documentation for '
                 '"global.cylc[platforms][localhost]" for more information.'
             )
-    # The list is reversed to allow user-set platforms (which are loaded
-    # later than site set platforms) to be matched first and override site
-    # defined platforms.
```

Comment:
Ditto

Reply:
Will do it for both if I do it for either :. resolving